### PR TITLE
Hotfix/video player

### PR DESCRIPTION
**Pull request by IvanSkadorva**

### CODE REVIEW

File: src/app/pages/video-player/video-dropdown-info/video-dropdown-info.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="dropdown-info__container">
     <video-back-button class="dropdown-info__back"></video-back-button>
     <h4 class="dropdown-info__tv-show-name">Black Mirror</h4>
-    <h2 class="dropdown-info__series-name">{{ currentVideo.title }}</h2>
+    <!--    <h2 class="dropdown-info__series-name">{{ currentVideo.title }}</h2>-->
```
Comment (marinbonibon) [inline src/app/pages/video-player/video-dropdown-info/video-dropdown-info.component.html:5]: commented code

File: src/app/services/video-player.service.ts [hunk 2]
```diff
@@ -89,4 +94,34 @@ export class VideoPlayerService {
...
+  }
+  public setCurrentTime(time: number): void {
+    this.currentTime.next(time);
+  }
+
+  public getVideoDuration(): number {
+    return this.videoDuration.value;
+  }
+  public setVideoDuration(duration: number): void {
+    this.videoDuration.next(duration);
```
Comment (marinbonibon) [inline src/app/services/video-player.service.ts:118]: Why do you need this?)

File: src/app/pages/video-player/progress-bar/progress-bar.component.ts [hunk 3]
```diff
@@ -1,28 +1,60 @@
...
+    this.initialX = this.playProgress.nativeElement.getBoundingClientRect().left;
+    this.dragItemWidth = this.playProgress.nativeElement.parentElement.getBoundingClientRect().width;
   }
 
   public updateProgressBar(): void {
-    this.timeline.nativeElement.value = this.currentTime / this.videoDuration;
-    this.timelineCurrentTime.nativeElement.style.width =
+    this.currentTime = this.videoPlayerService.getCurrentTime();
+    this.playProgress.nativeElement.style.width =
       Math.floor((this.currentTime / this.videoDuration) * 100) + '%';
```
[the remark this answers is not in the document]
Comment (IvanSkadorva) [reply src/app/pages/video-player/progress-bar/progress-bar.component.ts:32]: fixed

File: src/app/services/video-player.service.ts (hunk 2 above)
Comment (IvanSkadorva) [reply src/app/services/video-player.service.ts:118]: Hmm, when loading the video player, it sets the video duration

Comment (marinbonibon) [reply src/app/services/video-player.service.ts:118]: And why do you need BehaviorSubject for this?

**Review by kuzkov — APPROVED** [review]
Nice work

**Pull request merged**